android/java/org/chromium/chrome/browser/crypto_wallet/fragments/TxFragment.java

```diff
@@ -0,0 +1,107 @@
+/* Copyright (c) 2021 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.chromium.chrome.browser.crypto_wallet.fragments;
+
```

Comment:
these imports are not used
```
import android.annotation.SuppressLint;
import android.view.MotionEvent;
import androidx.recyclerview.widget.LinearLayoutManager;
import androidx.recyclerview.widget.RecyclerView;
import org.chromium.brave_wallet.mojom.TxData;
import org.chromium.brave_wallet.mojom.TxData1559;
```

Reply:
thanks, now I see, it is used indeed. 

In my tests it somehow was able to build even with these imports commented out.